pages/ShowDetailsPage.tsx

```diff
@@ -266,8 +272,8 @@ export default function ShowDetailsView({ show, isVisible, onClose }: ShowDetail
               <Text style={styles.sectionTitle}>Archives</Text>
               {archives.length > 0 ? (
                 archives
-                  .sort((a, b) => new Date(b.date).getTime() - new Date(a.date).getTime())
-                  .map((archive, index) => {
+                  .sort((a: Archive, b: Archive) => new Date(b.date).getTime() - new Date(a.date).getTime())
```

Comment:
if archives is typed already it _should_ be picking up the type for a/b - is this for type hints, or human hints? it doesn't hurt to keep it here & makes it more explicit so no change required - just curious if it was missing type info somehow

(I think archive/index below are especially readable, but also AOK to leave if preferred)